src/database/FirehoseConnector.js

```diff
@@ -171,7 +171,7 @@ class FirehoseConnector {
                 releaseNumber,
                 shard,
                 userRegion
-            },
```

Comment:
Are we trying to mitigate errors in production or just make the code more defensive in general?

I wonder if we should give the `browserInfo` field above the same treatment?

Reply:
this mitigates some actual errors, but being more defensive is a good idea, I'll add the same guard to browserInfo